pkg/application/service/factory/service_factory.go

```diff
@@ -88,15 +81,6 @@ func NewServiceFactory(options ...Option) *ServiceFactory {
 		log.Info(nil, fmt.Sprintf("configuring a new service factory with %d options", len(options)))
 	}
 
-	// default function to return an instance of Verification service
-	f.verificationServiceFunc = func(_ ...verificationservice.VerificationServiceOption) service.VerificationService {
-		return verificationservice.NewVerificationService(f.getContext(), f.verificationServiceOptions...)
-	}
-
-	f.signupServiceFunc = func(_ ...signupservice.SignupServiceOption) service.SignupService {
-		return signupservice.NewSignupService(f.getContext().Client(), f.signupServiceOptions...)
-	}
-
```

Comment:
this was the only place where `verificationServiceFunc` and `signupServiceFunc` were used. Also making ready for additional cleanup